.hooks/pre-commit

```diff
@@ -1,68 +0,0 @@
-#!/bin/bash
```

Comment:
I tried removing the pre-commit because the hook validation is catching multiple errors from other Packs (not ours, Digital Defense Pack passes validation), and thus prevents us from performing a commit.

I asked about this in the slack channel but no response.

I reverted back the pre-commit, but again I cannot make a commit due to the validation errors it's catching from other Packs.
Is there a way around this?

Reply:
able to perform commit. All changes were made.